<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RBManager</class>
 <widget class="QMainWindow" name="RBManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RBManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="openUSBManager">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Open Drive</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="fileList">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>591</width>
      <height>311</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="showDropIndicator" stdset="0">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="dragDropMode">
     <enum>QAbstractItemView::NoDragDrop</enum>
    </property>
    <property name="defaultDropAction">
     <enum>Qt::IgnoreAction</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
    <property name="rootIsDecorated">
     <bool>false</bool>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>581</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="searchBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="filterLabel">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Filter</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>281</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionOpen_File"/>
    <addaction name="actionCONFile_Info"/>
   </widget>
   <widget class="QMenu" name="menuDevice">
    <property name="title">
     <string>De&amp;vice</string>
    </property>
    <addaction name="actionDevice_Manager"/>
    <addaction name="actionUnmount_Drive"/>
    <addaction name="actionPoint_to_local_directory"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDevice"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen_File">
   <property name="text">
    <string>&amp;Import</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="actionDevice_Manager">
   <property name="text">
    <string>&amp;Device Manager</string>
   </property>
  </action>
  <action name="actionUnmount_Drive">
   <property name="text">
    <string>&amp;Unmount</string>
   </property>
  </action>
  <action name="actionPoint_to_local_directory">
   <property name="text">
    <string>&amp;Point to local directory</string>
   </property>
  </action>
  <action name="actionCONFile_Info">
   <property name="text">
    <string>CONFile Info</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>Info</string>
   </property>
  </action>
  <action name="actionSetHeaderArtist">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Artist</string>
   </property>
  </action>
  <action name="actionSetHeaderSongName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionSetHeaderAlbum">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Album</string>
   </property>
  </action>
  <action name="actionSetHeaderFilename">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </action>
  <action name="actionSetHeaderAuthor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Author</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
